books/RayTracingTheNextWeek.html

```diff
@@ -2252,9 +2249,6 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     class noise_texture : public texture {
       public:
-        noise_texture() {}
```

Comment:
For _production_ code, this should really be

    noise_texture() = delete;

but don't need it for this code.